analytics_data_api/v0/views/learners.py

```diff
@@ -60,6 +60,7 @@ class LearnerView(LastUpdateMixin, CourseViewMixin, generics.RetrieveAPIView):
         Returns metadata and engagement data for the learner in JSON format.
 
             * username: The username of the enrolled learner.
+            * account_url:  URL to learner's account page.
```

Comment:
I could see us doing it for the single learner endpoint, but then there's an asymmetry between that and the list endpoint. Maybe that's alright?